pages/test_analytics/importing_junit_xml.md.erb

```diff
@@ -4,6 +4,15 @@ While most test frameworks have a built-in JUnit XML export feature, these JUnit
 
 {:toc}
 
+## Mandatory JUnit XML Attributes
```

Comment:
H1 headings are the only headings to use title case, so this should be in sentence case ([style guide ref](https://github.com/buildkite/docs/blob/main/styleguide/STYLE.md#title-capitalization)).

```suggestion
## Mandatory JUnit XML attributes
```